### PR TITLE
Feature/30561 Pupil spa footer

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
- [x] Reintroduce footer component within only login component and add new components to map the links
- [x] Add the appropriate markup and content to the new components

### CODE REVIEW

File: pupil-spa/src/app/privacy/privacy.component.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p>
+  privacy works!
```
Comment (jon-shipley) [inline pupil-spa/src/app/privacy/privacy.component.html]: content?

File: pupil-spa/src/app/accessibility-statement/accessibility-statement.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<p>
+  accessibility-statement works!
```
[the remark this answers is not in the document]
Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/accessibility-statement/accessibility-statement.component.html]: @jon-shipley it was decided for this particular one to be added on a later PR

File: pupil-spa/src/app/contact/contact.component.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<p>
+  contact works!
```
[the remark this answers is not in the document]
Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/contact/contact.component.html]: @jon-shipley added

File: pupil-spa/src/app/privacy/privacy.component.html (hunk 1 above)
Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/privacy/privacy.component.html]: @jon-shipley added

**Pull request merged**